### DISCUSSION TITLE
download fail after last update

### DISCUSSION THREAD

**Discussion by sympan74** [aclap-dev/video-downloadhelper#1069] [Help and Q&A]
### Your question

i have problem to download from https://xxxgr.net/ after last update
return error

### Addon & CoApp version and details

Version 9.0.2.12 (stable), target google, locale el.

Η Συνοδός Εφαρμογή εγκαταστάθηκε (v2.0.19) Επανέλεγχος

**Comment by Mattmjs** [comment]
It looks to work on Firefox. when does it have an error ?

**Comment by sympan74** [comment]
attach error in print screen

[image: image.png]

**Comment by sympan74** [comment]
this is error text
Download failed


Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!).

**Reply by Mattmjs** [reply]
That sounds like it is happening immediately once the download starts. Check your anti virus & firewall are not blocking
anything. The companion app comes with .exe files which are erroneously flagged as a malware or viruses. These are false positives.
It also doesn't work using a proxy.

**Comment by sympan74** [comment]
check and tell you
thanks
